### PR TITLE
Checksum namespace

### PR DESCRIPTION
**Pull request by thibault-martinez**
#134

### CODE REVIEW

File: include/iota/crypto/checksum.hpp [hunk 1]
```diff
@@ -26,28 +26,20 @@
...
-public:
-  Checksum();
-  virtual ~Checksum();
+namespace Checksum {
 
-public:
-  Types::Trytes add(const Types::Trytes& address);
-  Types::Trytes remove(const Types::Trytes& address) const;
-  bool         isValid(const Types::Trytes& address);
+Types::Trytes add(const Types::Trytes& address);
```
Comment (Cylix) [inline include/iota/crypto/checksum.hpp]: can you add documentation for these functions? We can split it in another PR, but can be a good opportunity to improve doc coverage.

Comment (thibault-martinez) [reply include/iota/crypto/checksum.hpp]: Sure, it'll help for #101
